teerex/src/lib.rs

```diff
@@ -405,7 +406,7 @@ pub mod pallet {
 			signature: Vec<u8>,
 			certificate_chain: Vec<u8>,
 		) -> DispatchResultWithPostInfo {
-			log::info!("teerex: called into runtime call register_quoting_enclave()");
+			log::info!(target: TEEREX, "Called into runtime call register_quoting_enclave()");
```

Comment:
Don't you think most of these logs are trace?

Reply:
Absolutely, when I dropped the patches for the TCB Info V2 compatibility it seems I had dropped commits  I should have kept, sorry about this! :crying_cat_face: